clients/avro/src/main/java/io/confluent/examples/clients/basicavro/ConsumerExample.java

```diff
@@ -44,4 +60,15 @@ public static void main(final String[] args) {
 
         }
     }
+
+    public static Properties loadConfig(final String configFile) throws IOException {
+      if (!Files.exists(Paths.get(configFile))) {
+        throw new IOException(configFile + " not found.");
+      }
+      final Properties cfg = new Properties();
+      try (InputStream inputStream = new FileInputStream(configFile)) {
+        cfg.load(inputStream);
+      }
+      return cfg;
+    }
```

Comment:
As they are identical, I recommend we share this function across the two examples.  Since they are public and static, would be a simple matter of deleting one of them and adding an include for the other one's function.  LMK if you want me to make the change and push or I can review after you make it.

Reply:
@rspurgeon if you have the availability, it would be fantastic if you could verify & push the change.  If not, please let me know.

Reply:
Thinking about this more, while this suggestion achieves deduplication, the downside is:

1. It creates an unnecessary dependency between the two classes
2. It's not a logical dependency
3. Anyone copying one of the examples will encounter friction with this dependency

So I propose we leave it as it is

Reply:
Because it's compiled within the same project in the same namespace, IMO 1 & 2 aren't concerns. You just need a place to put a function, Ideally there would be a little file utility class to put it, but old school Java makes that annoying.  For 3. if the point is for someone to copy and paste the example from the docs or from this example then I agree make it easier for that.  Regardless, for this simple example I have no objection to in-lining the code or leaving the duplicate function.